src/audio/drc/drc.c

```diff
@@ -311,6 +311,27 @@ static void drc_params(struct processing_module *mod)
 }
 #endif /* CONFIG_IPC_MAJOR_4 */
 
+void *my_addr(void);
+
+void *drc_who_called(void)
+{
+	return my_addr();
+	//return __builtin_return_address(0);
```

Comment:
out of interest, why wont the builtin work ? Its not relying on linker script to set address is it ?

Reply:
@lgirdwood good question, I don't have an answer to it yet, it's still under investigation.

Reply:
builtin is probably set at build time now. I would bet that this is a relocatable ELF object, maybe with a different type/id that we have to fixup at runtime link time.